src/Microsoft.ML.TimeSeries/SequentialAnomalyDetectionTransformBase.cs

```diff
@@ -348,7 +348,7 @@ public DataViewSchema.DetachedColumn[] GetOutputColumns()
                         return col => false;
                 }
 
-                public void Save(ModelSaveContext ctx) => _parent.SaveModel(ctx);
+                void ICanSaveModel.Save(ModelSaveContext ctx) => _parent.SaveModel(ctx);
 
```

Comment:
This one was internal already, but in every other Mapper elsewhere we have made this one explicit despite it being internal for consistency.